serverless-local.yml

```diff
@@ -0,0 +1,391 @@
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: hathor-explorer-service
+useDotenv: true
+# The `provider` block defines where your service will be deployed
+provider:
+  name: aws
+  runtime: python3.8
+  region: ${env:AWS_DEFAULT_REGION}
+  lambdaHashingVersion: 20201221
+  environment:
+    HATHOR_CORE_DOMAIN: ${env:HATHOR_CORE_DOMAIN}
+    HATHOR_NODES: ${env:HATHOR_NODES}
+    REDIS_KEY_PREFIX: ${env:REDIS_KEY_PREFIX}
+    REDIS_HOST: ${env:REDIS_HOST}
+    REDIS_PORT: ${env:REDIS_PORT}
+    REDIS_DB: ${env:REDIS_DB}
+    METADATA_BUCKET: ${env:METADATA_BUCKET}
+    CORS_ALLOWED_REGEX: ${env:CORS_ALLOWED_REGEX}
+    NODE_CACHE_TTL: ${env:NODE_CACHE_TTL}
+
+plugins:
+  - serverless-s3-local
+  - serverless-offline
+  - serverless-python-requirements
+  - serverless-api-gateway-caching
+
+package:
+  individually: true
+  patterns:
+    - '!*'
+    - '!**/*'
+    - 'common/**'
+    - 'domain/**'
+    - 'gateways/**'
+    - 'usecases/**'
+    - 'utils/**'
+    - '!**/__pycache__'
+
+custom:
+  serverless-offline:
+    host: 0.0.0.0
+    httpPort: ${env:API_PORT}
+  pythonRequirements:
+    layer: true
+  s3:
+    host: localhost
+    directory: /tmp
+  apiGatewayCaching:
+    enabled: true
+    ttlInSeconds: 300
+
+functions:
+  get_node_handler:
+    handler: handlers/get_node_handler.handle
+    package:
+      patterns:
+        - 'handlers/get_node_handler.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: node/{hash}
+          method: get
+          cors: true
+          request:
+            parameters:
+              paths:
+                hash: true
+
+  list_nodes_handler:
+    handler: handlers/list_nodes_handler.handle
+    package:
+      patterns:
+        - 'handlers/list_nodes_handler.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: node/
+          method: get
+          cors: true
+
+  get_network_handler:
+    handler: handlers/get_network_handler.handle
+    package:
+      patterns:
+        - 'handlers/get_network_handler.py'
+    layers:
+      - { Ref: PythonRequirementsLambdaLayer }
+    events:
+      - http:
+          path: network/
+          method: get
+          cors: true
+
+  node_data_aggregator_handler:
```

Comment:
This Lambda is called by the Daemon based on its name in https://github.com/HathorNetwork/hathor-explorer-service/blob/7c90273201fc5390874c61b0eeb215f6d891efa4/gateways/node_gateway.py#L51

I saw that you used `.env.example` as env for the containers, and there we have:

```
LAMBDA_INVOKE_URL=http://localhost:3002/
DATA_AGGREGATOR_LAMBDA_NAME=hathor-explorer-service-dev-node_data_aggregator_handler
```

Did you test if this Lambda is correctly called by the daemon when running them in docker-compose?

I would assume it's not, because of the `localhost` address.